Rakefile

```diff
@@ -0,0 +1,50 @@
+CONTAINER_USERID = %x(id -u)
+
+namespace :docker do
+  desc 'Run your development environment'
+  task :up do
+    sh "docker-compose up"
+  end
+
+  desc 'Build your development environment'
+  task :build do
+    sh "docker build . -t openbuildservice/frontend --build-arg CONTAINER_USERID=#{CONTAINER_USERID}"
+    sh "contrib/bootstrap.rb"
+  end
+
+  desc 'Rebuild our static docker containers'
+  task :rebuild do
+    sh "docker build . -t openbuildservice/base:423 -t openbuildservice/base -f Dockerfile.423"
+    sh "docker build . -t openbuildservice/mariadb:423 -t openbuildservice/mariadb -f Dockerfile.mariadb"
+    sh "docker build . -t openbuildservice/memcached:423 -t openbuildservice/memcached -f Dockerfile.memcached"
+    sh "docker build . -t openbuildservice/backend:423 -t openbuildservice/backend -f Dockerfile.backend"
+  end
+
+  desc 'Publish our docker containers'
+  task publish: [:rebuild] do
+    sh "docker push openbuildservice/base:423"
+    sh "docker push openbuildservice/base"
+    sh "docker push openbuildservice/mariadb:423"
+    sh "docker push openbuildservice/mariadb"
+    sh "docker push openbuildservice/memcached:423"
+    sh "docker push openbuildservice/memcached"
+    sh "docker push openbuildservice/backend:423"
+    sh "docker push openbuildservice/backend"
+  end
+
+  desc 'Attach to the docker container'
+  task :attach do
+    sh "docker-compose exec frontend /bin/bash -l"
+  end
+
+  namespace :test do
+    desc 'Run our frontend tests in the docker container'
+    task :frontend do
+      sh "docker-compose exec frontend bundle exec rspec"
+    end
+    desc 'Run our backend tests in the docker container'
+    task :backend do
+      sh "docker-compose exec backend cd /obs; make -C src/backend test"
```

Comment:
I think you meant:

`sh "docker-compose exec backend bash -c 'cd /obs; make -C src/backend test'"`